src/vs/workbench/electron-sandbox/splash.ts

```diff
@@ -73,16 +67,22 @@ class PartsSplash {
 	}
 
 	private _savePartsSplash() {
-		const baseTheme = getThemeTypeSelector(this._themeService.getColorTheme().type);
+		this._savePartsSplashScheduler.trigger(() => this._doSavePartsSplash(), 800);
+	}
```

Comment:
I don't believe that the `ThrottledDelayer` is an improvement. I see more code and more allocations

Reply:
Removed it.